### PR TITLE
ART-6671 Trigger microshift_sync

### PR DESCRIPTION
**Pull request by locriandev**
Ref. https://issues.redhat.com/browse/ART-6671

### CODE REVIEW

File: pyartcd/pyartcd/jenkins.py [hunk 1]
```diff
@@ -221,3 +222,14 @@ def start_sync_for_ci(version: str, blocking: bool = False):
         },
         blocking=blocking
     )
+
+
+def start_microshift_sync(version: str, assembly: str, blocking: bool = False):
```
Comment (thegreyd) [inline pyartcd/pyartcd/jenkins.py:227]: Is there a dry_run? It might be worth it to have a dry_run

**Comment by thegreyd** [comment]
A test job in hackspace would be nice to verify this doesn't break on anything

File: pyartcd/pyartcd/jenkins.py (hunk 1 above)
Comment (locriandev) [reply pyartcd/pyartcd/jenkins.py:227]: The job has a `DRY_RUN`, but it's not needed when triggered by automation. You can always use it when starting a run by hand, but the Python interface won't need it

**Comment by locriandev** [comment]
> A test job in hackspace would be nice to verify this doesn't break on anything

I agree, but it's a simple change and it's mostly about having it run in production. Possibly, a fix will be easy (same for a revert in case or emergencies)

**Comment by locriandev** [comment]
> @locriandev If you have the jira ticket in title, the PR would get auto-linked to that ticket, fyi

Does not seem to be working though

File: pyartcd/pyartcd/pipelines/build_microshift.py [hunk 2]
```diff
@@ -141,7 +142,9 @@ async def run(self):
             if pr:
                 message += f"\nA PR to update the assembly definition has been created/updated: {pr.html_url}"
             if assembly_type in [AssemblyTypes.PREVIEW, AssemblyTypes.CANDIDATE]:
-                message += f"\n This is a {assembly_type.name} release. Please run <https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/aos-cd-builds/job/build%252Fmicroshift_sync/|microshift_sync> to publish the build to mirror."
+                jenkins.start_microshift_sync(version=version, assembly=self.assembly)
```
Comment (vfreex) [inline pyartcd/pyartcd/pipelines/build_microshift.py:145]: The `BUILD_VERSION` parameter expects `<major>.<minor>`, like `4.14`. However the `version` returned by `generate_microshift_version_release(release_name)` is the rpm version, which includes the `.z` part, like `4.14.1`.

Let's use value from `major, minor = util.isolate_major_minor_in_group(self.group)` instead.

**Review by vfreex — COMMENTED** [review]
One issue, otherwise LGTM

File: pyartcd/pyartcd/pipelines/build_microshift.py (hunk 2 above)
Comment (locriandev) [reply pyartcd/pyartcd/pipelines/build_microshift.py:147]: Fixed in https://github.com/openshift-eng/aos-cd-jobs/pull/3820/commits/fcbbe7322089d052618b9ccaf065e5f4537b738a

**Pull request merged**